pkg/storage/db.go

```diff
@@ -437,6 +437,16 @@ func (db *DB) StoreV2(ctx context.Context, cloudAssetChanges domain.CloudAssetCh
 					}
 				}
 			}
+			for _, res := range val.RelatedResources {
```

Comment:
info: non-blocking concern. This function is getting rather large at 60 lines and It has quite a bit of loops that could be separated into their own functions.

Scope wise I don't think it needs to change here since it was like this already but for readabilities sake we should probably break this function up in the near future.

Reply:
I agree, there is lot of room for refactor and improving code quality in db.go